docs/user-guide/subset.ipynb

```diff
@@ -131,6 +131,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Let's add a user-facing property for this. Accessing the hidden attributes is not a best practice.</p><p> </p><p><code>Grid.is_subset</code> which will access <code>Grid._is_subset</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/1122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>